feedthefox/users/urls.py

```diff
@@ -3,7 +3,9 @@
 from django.conf.urls import include, url
 
 from allauth.account import views as account_views
-from allauth.socialaccount import providers, views as socialaccount_views
+from allauth.socialaccount import providers
+
+from feedthefox.users import views
```

Comment:
The reason I commented here and not in the other one, it's because it makes sense to use the project's views as default and import other libraries as `library_views`.